### PR TITLE
Improve Aztec media "loading in progress" icons

### PR DESCRIPTION
**Pull request by daniloercoli**
This PR adds & improves support for media placeholder(s):
- Add media placeholder to `AztecImageLoader` & `AztecVideoLoader`, so that a placeholder is shown on the screen while the pictures is loading from the Network. [To check this open a post with pictures in it. When the conversion *fromHTML* to Aztec is done pictures are loaded from the network]

-  Add media placeholder to ` AztecEditorFragment` when a "network" pictures is added to the editor. For "network" picture I mean a picture already available in the WP Media Library. In this case the URL of the picture is passed to Aztec, a placeholder is shown until the picture is fully loaded from the Network.

In absence of these placeholders when you add a picture from the network the Editor doesn't show anything, and it's looks like that you've not added anything to it. Until the picture is fully loaded from the network.

Note: Not sure if it's OK or not, but I've linked Aztec develop in this PR.

cc @0nko @mzorz

### CODE REVIEW

**Comment by daniloercoli** [comment]
cc @mzorz Would you mind to review this one? Since i've fixed some merging conflicts in `AztecEditorFragment.java`, so it's better that you recheck it.

File: libs/editor/WordPressEditor/build.gradle [hunk 1]
```diff
@@ -51,8 +51,8 @@ dependencies {
     compile 'com.android.support:design:25.3.1'
     compile 'org.apache.commons:commons-lang3:3.5'
     compile 'org.wordpress:utils:1.16.0'
-    compile ('com.github.wordpress-mobile.AztecEditor-Android:aztec:v1.0-beta.6')
-    compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:wordpress-comments:v1.0-beta.6')
+    compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:aztec:develop-SNAPSHOT')
+    compile ('com.github.wordpress-mobile.WordPress-Aztec-Android:wordpress-comments:develop-SNAPSHOT')
```
Comment (mzorz) [inline libs/editor/WordPressEditor/build.gradle]: Can we revert this change here?

**Comment by 0nko** [comment]
We probably shouldn't use `develop` Aztec dependency reference in wpandroid, since it can change anytime. Even though you need to manually update the cache to get the latest version, someone else could get a different library.

**Comment by mzorz** [comment]
> We probably shouldn't use develop-SNAPSHOT Aztec dependency reference in wpandroid, 

:D totally,  commented here https://github.com/wordpress-mobile/WordPress-Android/pull/6433#discussion_r131162421.

We're adding the hash here as Danilo tells me this PR needs some of the latest stuff in Aztec, and you'll be able to re-reference the next beta tag once it's ready

**Comment by mzorz** [comment]
ok tested! :shipit:
